tests/testthat/test-truncate_triangles.R

```diff
@@ -30,7 +30,7 @@ test_that("truncate_triangles returns correct number of truncated matrices with
 })
 
 test_that("truncate_triangles default n calculation works correctly", {
```

Comment:
need an explicit test for the edge case where it uses the min

Reply:
will add

Reply:
Have removed the minimum so it is just one set value now